Add table aws_securityhub_finding_aggregator closes #1002 (#1031)
turbot/steampipe-plugin-aws: docs/tables/aws_securityhub_finding_aggregator.md

# Table: aws_securityhub_finding_aggregator

AWS Security Hub now allows you to designate an aggregation Region and link some or all Regions to that aggregation Region. This gives you a centralized view of all your findings across all of your accounts and all of your linked Regions.

## Examples

### Basic info

```sql
select
  arn,
  finding_aggregation_region,
  region_linking_mode
from
  aws_securityhub_finding_aggregator;
```

> ### List finding aggregators with region linking mode ALL_REGIONS 

Reviewer: ```suggestion
### List finding aggregators linked to all regions
```

```sql
select
  arn,
  finding_aggregation_region,
  region_linking_mode
from
  aws_securityhub_finding_aggregator
where
  region_linking_mode = 'ALL_REGIONS';
```

> ### List regions for finding aggregators with linking mode SPECIFIED_REGIONS

Reviewer: ```suggestion
### List regions for finding aggregators that include specific regions
```

Does this also cover finding aggregators that exclude certain regions? If not, can you please include an example for listing those as well?
